src/listeners/client/error.ts

```diff
@@ -1,8 +1,8 @@
 import { ApplyOptions } from '@sapphire/decorators';
 import { Events, Listener } from '@sapphire/framework';
 import * as Sentry from '@sentry/node';
-import { logErrorToContainer } from '../../lib/utils/errorHandling';
 import { envIsDefined } from '@skyra/env-utilities';
+import { logErrorToContainer } from '../../lib/utils/functions/errorHandling';
```

Comment:
```suggestion
import { logErrorToContainer } from '#utils/functions/errorHandling';
```

Reply:
I found a bunch of them, you probably will fix all of them later on so just ignore this :)